graphene_django/utils/testing.py

```diff
@@ -24,7 +24,7 @@ def setUpClass(cls):
 
         cls._client = Client()
 
-    def query(self, query, op_name=None, input_data=None):
+    def query(self, query, op_name=None, input_data=None, variables=None):
```

Comment:
Could you update the docstring below with the new argument?

Reply:
thanks, updated.